linkerd/identity/src/lib.rs

```diff
@@ -51,6 +51,10 @@ struct CertResolver(rustls::sign::CertifiedKey);
 #[derive(Clone, Debug)]
 pub struct InvalidCrt(rustls::TLSError);
 
+/// A marker type for local server identities.
```

Comment:
nit: i wouldn't call this a "marker" type --- I think in Rust, "marker" is typically used to refer to traits like `Copy`, `Send`, `Sync` etc which signify something about a type without specifying methods/behaviors.